composer.json

```diff
@@ -16,51 +16,52 @@
         "php": ">=8.1",
         "ext-pdo_sqlite": "*",
         "doctrine/dbal": "^3.1",
-        "doctrine/doctrine-bundle": "^2.8",
-        "doctrine/doctrine-migrations-bundle": "^3.0",
-        "doctrine/orm": "^2.11",
+        "doctrine/doctrine-bundle": "2.11.x-dev",
+        "doctrine/doctrine-migrations-bundle": "3.3.x-dev",
```

Comment:
3.3.x-dev is not the latest version. why requiring that version ?

Reply:
I thought it was the most recent and updated branch in that repo: https://github.com/doctrine/DoctrineMigrationsBundle/branches

Reply:
ah sorry. I messed up between doctrine bundles